### PR TITLE
Update browser-bugs.md

### LINKED ISSUE #30717: Add a warning callout in browser bugs page
In v4 we still have the Browser Bugs page, but it's outdated for sure. We should add a callout or something. We could also remove the page but maybe it makes more sense to keep it with a warning at the top?

/CC @mdo @MartijnCuppens

### PR DESCRIPTION
**Pull request by XhmikosR**
Add a danger callout that the page is outdated

/CC @mdo @MartijnCuppens 

Fixes #30717 

Preview: <https://deploy-preview-30751--twbs-bootstrap.netlify.app/docs/4.4/browser-bugs/>

### CODE REVIEW

File: site/docs/4.4/browser-bugs.md [hunk 1]
```diff
@@ -5,6 +5,13 @@ group: browser-bugs
 redirect_from: "/browser-bugs/"
 ---
 
+{% capture callout %}
+##### The Page is no longer maintained
```
Comment (mdo) [inline site/docs/4.4/browser-bugs.md:9]:
```suggestion
##### Outdated content
```

File: site/docs/4.4/browser-bugs.md [hunk 2]
```diff
@@ -5,6 +5,13 @@ group: browser-bugs
...
+
+"The Wall of browser bugs" page is outdated for many versions now and is only kept for historical reasons.
```
Comment (mdo) [inline site/docs/4.4/browser-bugs.md:11]:
```suggestion
This page is outdated and is no longer applicable to the latest versions of Bootstrap. It's here purely for historical purposes now and will be removed in our next major release.
```

**Review by mdo — COMMENTED** [review]
Two suggested edits

**Comment by XhmikosR** [comment]
Updated @mdo

**Review by mdo — APPROVED** [review]

**Pull request merged**